docs/configuration/http_conn_man/route_config/weighted_routing.rst

```diff
@@ -0,0 +1,109 @@
+.. _config_http_conn_man_route_table_weighted_routing:
+
+Traffic splitting across multiple upstreams
+===========================================
+
+Envoy's router can split traffic to a route in a virtual host across
+multiple upstream clusters. A common use case is A/B testing or
+multivariate testing, where two or more versions of the same service are
+tested simultaneously. In this case, the traffic to the route has to be
+*split* between clusters running different versions of the same
+service.
+
+Consider a simple example ``service`` with three versions (v1, v2 and
+v3). Envoy provides two ways to split traffic evenly across the three
+versions (i.e., ``33%, 33%, 34%``):
+
+.. _config_http_conn_man_route_table_weighted_routing_percentages:
+
+(a) Weight-based cluster selection
+----------------------------------
+
+The first option is to use a **single** :ref:`route <config_http_conn_man_route_table_route>` with 
+:ref:`weighted_clusters <config_http_conn_man_route_table_route_weighted_clusters>`,
+where multiple upstream cluster targets are specified for a single route,
+along with weights that indicate the **percentage** of traffic to be sent
+to each upstream cluster.
+
+.. code-block:: json
+
+    {
+      "route_config": {
+        "virtual_hosts": [
+          {
+            "name": "service",
+            "domains": ["*"],
+            "routes": [
+              {
+                "prefix": "/",
+                "weighted_clusters": {
+                  "clusters" : [
+                    { "name" : "service_v1", "weight" : 33 },
+                    { "name" : "service_v2", "weight" : 33 },
+                    { "name" : "service_v3", "weight" : 34 }
+                  ]
+                }
+              }
+            ]
+          }
+        ]
+      }
+    }
+
+.. _config_http_conn_man_route_table_weighted_routing_probabilities:
+
+(b) Probabilistic route selection
```

Comment:
I think I would kill traffic shifting in place of your new docs. @junr03 thoughts? You wrote the other doc.